specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-04-25-preview/examples/Machines_ListBySubscription.json

```diff
@@ -0,0 +1,155 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "api-version": "2023-04-25-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.HybridCompute/machines/myMachine",
+            "name": "myMachine",
+            "location": "eastus2euap",
+            "tags": null,
+            "identity": {
+              "type": "SystemAssigned",
+              "principalId": "string",
+              "tenantId": "string"
+            },
+            "type": "Microsoft.HybridCompute/machines",
+            "kind": null,
+            "properties": {
+              "provisioningState": "Succeeded",
+              "agentVersion": null,
+              "status": null,
+              "lastStatusChange": null,
+              "errorDetails": null,
+              "displayName": null,
+              "machineFqdn": null,
+              "vmUuid": null,
+              "osSku": null,
+              "domainName": null,
+              "adFqdn": null,
+              "dnsFqdn": null,
+              "osVersion": null,
+              "osType": null,
+              "osProfile": {
```

Comment:
The examples look incomplete. Are these properties actually supposed to be nullable?

Reply:
Yes, a lot of these properties are filled in specific scenarios, for example the kind property is filled in when the machine is onboarded in a private cloud scenario, the version of the agent installed on the machine also affects which properties get filled